cluster/manifests/skipper/daemonset.yaml

```diff
@@ -51,6 +51,7 @@ spec:
           - "-kubernetes-path-mode=path-prefix"
           - "-address=:9999"
           - "-wait-first-route-load"
+          - "-enable-kubernetes-east-west"
```

Comment:
Is it too late at this point to ask for a more intuitive name for this?

Reply:
yes, because a change would be a breaking change